compiler/z/codegen/OMRTreeEvaluator.cpp

```diff
@@ -5984,7 +5949,7 @@ OMR::Z::TreeEvaluator::lloadEvaluator(TR::Node * node, TR::CodeGenerator * cg)
 
 /**
  * sload Evaluator: load short integer
- *   - also handles cload, isload and icload
+ *   - also handles isload and icload
```

Comment:
Is this comment right? isn't cloadi being eliminated?

Reply:
Fixed in d89a0b3